gflownet/gflownet.py

```diff
@@ -984,6 +947,10 @@ def log_iter(
 
 class Policy:
     def __init__(self, config, env, device, float_precision, base=None):
+        # If config is null, default to uniform
+        if config is None:
+            config = OmegaConf.create()
+            config.type = "uniform"
```

Comment:
That's a minor concern, but it seems to me it would be better to set it to "fixed" and let it be defined by the environment instead of being "uniform" because for some envs "uniform" may not exists (e.g. unbounded continuous spaces). Related thought: "uniform" is essentially a specific case of "fixed", maybe it should be moved into corresponding environments?

Reply:
Yeah I had thought of whether it should be uniform or fixed... For the purposes here it's better if it's random (i.e. uniform) than fixed and I agree it might not be defined in some special cases, but we don't need to deal with that at the moment.